notebooks/idr0062_prediction.ipynb

```diff
@@ -476,14 +467,76 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 13,
+   "execution_count": 15,
    "id": "interior-summit",
    "metadata": {},
    "outputs": [],
    "source": [
     "conn.close()"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "4d4fac89",
+   "metadata": {},
+   "source": [
+    "## Save the StarDist labels\n",
+    "\n",
+    "Stardist offers method to save the labels into ImageJ rois using ``export_imagej_rois``. This is outside the scope of this notebook. \n",
+    "\n",
+    "Below we show how to save the labels locally in a machine and human readable format: **geojson**.\n",
+    "\n",
+    "* Convert the Stardist labels into geojson Polygons\n",
```

Comment:
```suggestion
    "* Convert the Stardist polygon coordinates into geojson Polygons\n",
```

I think you are not using the label image here, just the ``results_details`` which are described as polygon details above.